obci/interfaces/bci/budzik/experimental/budzik_prototype.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from obci.configs import settings
+from obci.interfaces.bci.analysis_master import AnalysisMaster
+from obci.interfaces.bci.budzik.experimental.dummy_classifier import DummyClassifier
+from obci.analysis.buffers.auto_ring_buffer import AutoRingBuffer
+
+
+class BudzikPrototype(AnalysisMaster):
```

Comment:
Żadne? Decyzje otrzymuje funkcja add_result, AnalysisMastera.

On 14 April 2016 at 16:50, Anna Chabuda <notifications@github.com> wrote:

> In obci/interfaces/bci/budzik/experimental/budzik_prototype.py
> <https://github.com/BrainTech/openbci/pull/5#discussion_r59730902>:
>
> > @@ -0,0 +1,42 @@
> > +#!/usr/bin/env python
> > +# -*- coding: utf-8 -*-
> > +
> > +from obci.configs import settings
> > +from obci.interfaces.bci.analysis_master import AnalysisMaster
> > +from obci.interfaces.bci.budzik.experimental.dummy_classifier import DummyClassifier
> > +from obci.analysis.buffers.auto_ring_buffer import AutoRingBuffer
> > +
> > +
> > +class BudzikPrototype(AnalysisMaster):
>
> to jakie decyzje przesyła twój prototyp? ;)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/BrainTech/openbci/pull/5/files/30fc63370bc28fee2a40098aec3e010f3e61595b#r59730902>
>